frontend/src/components/videoPlayer/VideoPlayer.js

```diff
@@ -24,21 +25,27 @@ const useStyles = makeStyles(() => ({
   },
 }));
 
-const VideoPlayer = () => {
+const VideoPlayer = (data) => {
   // eslint-disable-next-line max-len
+  const [movie, setMovie] = useState({});
   const classes = useStyles();
   const src = Vid;
 
+  useEffect(async () => {
+    const movieData = await movieService.getMovieData(data.movie.imdbCode);
```

Comment:
I think you need a try-catch block here to catch server errors. Tatiana has a good example in MyProfile :)

Reply:
Alright I will check that out in the evening and changed it! :)